website/content/en/docs/tasks/deprov-nodes.md

```diff
@@ -1,36 +1,84 @@
 ---
-title: "Deprovisioning nodes"
```

Comment:
I think you might need a rebase.

Reply:
Yeah. I know there have been some changes to this file. I'll do that.